\documentclass[10pt, letterpaper]{article}

% Packages:
\usepackage[
    ignoreheadfoot, % set margins without considering header and footer
    top=2 cm, % seperation between body and page edge from the top
    bottom=2 cm, % seperation between body and page edge from the bottom
    left=2 cm, % seperation between body and page edge from the left
    right=2 cm, % seperation between body and page edge from the right
    footskip=1.0 cm, % seperation between body and footer
    % showframe % for debugging 
]{geometry} % for adjusting page geometry
\usepackage[explicit]{titlesec} % for customizing section titles
\usepackage{tabularx} % for making tables with fixed width columns
\usepackage{array} % tabularx requires this
\usepackage[dvipsnames]{xcolor} % for coloring text
\definecolor{primaryColor}{RGB}{0, 79, 144} % define primary color
\usepackage{enumitem} % for customizing lists
\usepackage{fontawesome5} % for using icons
\usepackage{amsmath} % for math
\usepackage[
    pdftitle={Dr Javier Sánchez Utgés's CV},
    pdfauthor={Dr Javier Sánchez Utgés},
    pdfcreator={LaTeX with RenderCV},
    colorlinks=true,
    urlcolor=primaryColor
]{hyperref} % for links, metadata and bookmarks
\usepackage[pscoord]{eso-pic} % for floating text on the page
\usepackage{calc} % for calculating lengths
\usepackage{bookmark} % for bookmarks
\usepackage{lastpage} % for getting the total number of pages
\usepackage{changepage} % for one column entries (adjustwidth environment)
\usepackage{paracol} % for two and three column entries
\usepackage{ifthen} % for conditional statements
\usepackage{needspace} % for avoiding page brake right after the section title
\usepackage{iftex} % check if engine is pdflatex, xetex or luatex

% Ensure that generate pdf is machine readable/ATS parsable:
\ifPDFTeX
    \input{glyphtounicode}
    \pdfgentounicode=1
    \usepackage[T1]{fontenc}
    \usepackage[utf8]{inputenc}
    \usepackage{lmodern}
\fi

\usepackage[default, type1]{sourcesanspro} 

% Some settings:
\AtBeginEnvironment{adjustwidth}{\partopsep0pt} % remove space before adjustwidth environment
\pagestyle{empty} % no header or footer
\setcounter{secnumdepth}{0} % no section numbering
\setlength{\parindent}{0pt} % no indentation
\setlength{\topskip}{0pt} % no top skip
\setlength{\columnsep}{0.15cm} % set column seperation
\makeatletter
\let\ps@customFooterStyle\ps@plain % Copy the plain style to customFooterStyle
\patchcmd{\ps@customFooterStyle}{\thepage}{
    \color{gray}\textit{\small Dr Javier Sánchez Utgés - Page \thepage{} of \pageref*{LastPage}}
}{}{} % replace number by desired string
\makeatother
\pagestyle{customFooterStyle}

\titleformat{\section}{
    % avoid page braking right after the section title
    \needspace{4\baselineskip}
    % make the font size of the section title large and color it with the primary color
    \Large\color{primaryColor}
}{
}{
}{
    % print bold title, give 0.15 cm space and draw a line of 0.8 pt thickness
    % from the end of the title to the end of the body
    \textbf{#1}\hspace{0.15cm}\titlerule[0.8pt]\hspace{-0.1cm}
}[] % section title formatting

\titlespacing{\section}{
    % left space:
    -1pt
}{
    % top space:
    0.3 cm
}{
    % bottom space:
    0.2 cm
} % section title spacing

% \renewcommand\labelitemi{$\vcenter{\hbox{\small$\bullet$}}$} % custom bullet points
\newenvironment{highlights}{
    \begin{itemize}[
        topsep=0.10 cm,
        parsep=0.10 cm,
        partopsep=0pt,
        itemsep=0pt,
        leftmargin=0.4 cm + 10pt
    ]
}{
    \end{itemize}
} % new environment for highlights

\newenvironment{highlightsforbulletentries}{
    \begin{itemize}[
        topsep=0.10 cm,
        parsep=0.10 cm,
        partopsep=0pt,
        itemsep=0pt,
        leftmargin=10pt
    ]
}{
    \end{itemize}
} % new environment for highlights for bullet entries


\newenvironment{onecolentry}{
    \begin{adjustwidth}{
        0.2 cm + 0.00001 cm
    }{
        0.2 cm + 0.00001 cm
    }
}{
    \end{adjustwidth}
} % new environment for one column entries

\newenvironment{twocolentry}[2][]{
    \onecolentry
    \def\secondColumn{#2}
    \setcolumnwidth{\fill, 4.5 cm}
    \begin{paracol}{2}
}{
    \switchcolumn \raggedleft \secondColumn
    \end{paracol}
    \endonecolentry
} % new environment for two column entries

\newenvironment{threecolentry}[3][]{
    \onecolentry
    \def\thirdColumn{#3}
    \setcolumnwidth{1 cm, \fill, 4.5 cm}
    \begin{paracol}{3}
    {\raggedright #2} \switchcolumn
}{
    \switchcolumn \raggedleft \thirdColumn
    \end{paracol}
    \endonecolentry
} % new environment for three column entries

\newenvironment{header}{
    \setlength{\topsep}{0pt}\par\kern\topsep\centering\color{primaryColor}\linespread{1.5}
}{
    \par\kern\topsep
} % new environment for the header

\newcommand{\placelastupdatedtext}{% \placetextbox{<horizontal pos>}{<vertical pos>}{<stuff>}
  \AddToShipoutPictureFG*{% Add <stuff> to current page foreground
    \put(
        \LenToUnit{\paperwidth-2 cm-0.2 cm+0.05cm},
        \LenToUnit{\paperheight-1.0 cm}
    ){\vtop{{\null}\makebox[0pt][c]{
        \small\color{gray}\textit{Last updated in May 2025}\hspace{\widthof{Last updated in May 2025}}
    }}}%
  }%
}%

% save the original href command in a new command:
\let\hrefWithoutArrow\href

% new command for external links:
\renewcommand{\href}[2]{\hrefWithoutArrow{#1}{\ifthenelse{\equal{#2}{}}{ }{#2 }\raisebox{.15ex}{\footnotesize \faExternalLink*}}}


\begin{document}
    \newcommand{\AND}{\unskip
        \cleaders\copy\ANDbox\hskip\wd\ANDbox
        \ignorespaces
    }
    \newsavebox\ANDbox
    \sbox\ANDbox{}

    \placelastupdatedtext
    \begin{header}
        \fontsize{30 pt}{30 pt}
        \textbf{Dr Javier Sánchez Utgés}

        \vspace{0.3 cm}

        \normalsize
        \mbox{\hrefWithoutArrow{mailto:utges03@gmail.com}{{\footnotesize\faEnvelope[regular]}\hspace*{0.13cm}utges03@gmail.com}}%
        \kern 0.25 cm%
        \AND%
        \kern 0.25 cm%
        \mbox{\hrefWithoutArrow{tel:+44-7878-459945}{{\footnotesize\faPhone*}\hspace*{0.13cm}07878 459945}}%
        \kern 0.25 cm%
        \AND%
        \kern 0.25 cm%
        \mbox{\hrefWithoutArrow{https://orcid.org/0000-0002-1078-1440}{{\footnotesize\faOrcid}\hspace*{0.13cm}ORCID}}%
        \kern 0.25 cm%
        \AND%
        \kern 0.25 cm%
        \mbox{\hrefWithoutArrow{https://www.linkedin.com/in/javier-sánchez-utgés-6aa2961b3/}{{\footnotesize\faLinkedinIn}\hspace*{0.13cm}LinkedIn}}%
        \kern 0.25 cm%
        \AND%
        \kern 0.25 cm%
        \mbox{\hrefWithoutArrow{https://github.com/JavierSanchez-Utges}{{\footnotesize\faGithub}\hspace*{0.13cm}GitHub}}%
        \kern 0.25 cm%
        \AND%
        \kern 0.25 cm%
        \mbox{\hrefWithoutArrow{https://scholar.google.com/citations?user=Javier S Utgés}{{\footnotesize\faGraduationCap}\hspace*{0.13cm}Google Scholar}}%
    \end{header}

    \vspace{0.3 cm - 0.3 cm}


    \section{Education}



        
        \begin{threecolentry}{\textbf{PhD}}{
            Sept 2020 – Mar 2025
        }
            \textbf{University of Dundee}, Structural Bioinformatics
            \begin{highlights}
                \item Supervised 7 students across BSc, MSc and PhD degrees
                \item Student Representative of Computational Biology Division
                \item Thesis: \textit{"Computational methods for the characterisation and evaluation of protein-ligand binding sites"}
            \end{highlights}
        \end{threecolentry}

        \vspace{0.2 cm}

        \begin{threecolentry}{\textbf{MSc}}{
            Sept 2018 – July 2020
        }
            \textbf{Universitat Pompeu Fabra}, Bioinformatics for Health Sciences
            \begin{highlights}
                \item Grade: 9.3/10
                \item 9-month Internship at the \href{https://www.compbio.dundee.ac.uk/}{Barton Group}, University of Dundee
                \item Thesis: \textit{"Ankyrin repeats in context with population variation"}
            \end{highlights}
        \end{threecolentry}

        \vspace{0.2 cm}

        \begin{threecolentry}{\textbf{BSc}}{
            Sept 2014 – July 2018
        }
            \textbf{Universitat Autònoma de Barcelona}, Genetics
            \begin{highlights}
                \item Grade: 8.4/10
                \item 12-month Erasmus exchange at University of Dundee
                \item Dissertation: \textit{“Methods for the detection and estimation of selection acting upon the genome”}
            \end{highlights}
        \end{threecolentry}


    
    \section{Experience}



        
        \begin{twocolentry}{
            Dundee, Scotland

        Mar 2025 – June 2025
        }
            \textbf{University of Dundee}, Research assistant
        \end{twocolentry}


        \vspace{0.2 cm}

        \begin{twocolentry}{
            London, England

        Aug 2022 – Nov 2022
        }
            \textbf{The Association of the British Pharmaceutical Industry (ABPI)}, Intern
            \begin{highlights}
                \item Improved the Interactive Resources for Schools website of the \href{https://www.abpi.org.uk/}{ABPI}
            \end{highlights}
        \end{twocolentry}


        \vspace{0.2 cm}

        \begin{twocolentry}{
            Barcelona, Spain

        July 2017 – Sept 2017
        }
            \textbf{IBE-CSIC-UPF}, Intern
            \begin{highlights}
                \item Internship at the \href{https://www.ibe.upf-csic.es/calafell}{Calafell Lab} on Forensic and Population Genetics
            \end{highlights}
        \end{twocolentry}



    
    \section{Publications}



        
        \begin{samepage}
            \begin{twocolentry}{
                May 2025
            }
                \textbf{LIGYSIS-web: a resource for the analysis of protein-ligand binding sites}

                \vspace{0.10 cm}

                \mbox{\textbf{\textit{Utgés JS}}}, \mbox{MacGowan SA}, \mbox{Barton GJ}
                \vspace{0.10 cm}

        \href{https://doi.org/10.1093/nar/gkaf411}{10.1093/nar/gkaf411}
         (Nucleic Acids Research)    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Nov 2024
            }
                \textbf{Comparative evaluation of methods for the prediction of protein-ligand binding sites}

                \vspace{0.10 cm}

                \mbox{\textbf{\textit{Utgés JS}}}, \mbox{Barton GJ}
                \vspace{0.10 cm}

        \href{https://doi.org/10.1186/s13321-024-00923-z}{10.1186/s13321-024-00923-z}
         (Journal of Cheminformatics)    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Mar 2024
            }
                \textbf{Classification of likely functional class for ligand binding sites identified from fragment screening}

                \vspace{0.10 cm}

                \mbox{\textbf{\textit{Utgés JS}}}, \mbox{MacGowan SA}, \mbox{Ives CM}, \mbox{Barton GJ}
                \vspace{0.10 cm}

        \href{https://doi.org/10.1038/s42003-024-05970-8}{10.1038/s42003-024-05970-8}
         (Communications Biology)    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Aug 2021
            }
                \textbf{Ankyrin repeats in context with human population variation}

                \vspace{0.10 cm}

                \mbox{\textbf{\textit{Utgés JS}}}, \mbox{Tsenkov MI}, \mbox{Dietrich NJM}, \mbox{MacGowan SA}, \mbox{Barton GJ}
                \vspace{0.10 cm}

        \href{https://doi.org/10.1371/journal.pcbi.1009335}{10.1371/journal.pcbi.1009335}
         (PLOS Computational Biology)    \end{twocolentry}
        \end{samepage}


    
    \section{Projects}



        
        \begin{twocolentry}{
            \href{https://github.com/JavierSanchez-Utges/LIGYSIS-web}{LIGYSIS-web}
        }
            \textbf{LIGYSIS-web}
            \begin{highlights}
                \item A Python Flask Web Application to explore protein-ligand binding sites
                \item Tools: Python, JavaScript, HTML, CSS
                \item Packages: Flask, 3DMol.js, Chart.js
            \end{highlights}
        \end{twocolentry}


        \vspace{0.2 cm}

        \begin{twocolentry}{
            \href{https://github.com/bartongroup/LBS-comparison}{LBS-comparison}
        }
            \textbf{LBS-comparison}
            \begin{highlights}
                \item The largest benchmark of ligand binding site prediction methods.
                \item Tools: Python, P2Rank, fpocket, IF-SitePred, POVME
                \item Packages: BioPython, Pandas, Sklearn
            \end{highlights}
        \end{twocolentry}


        \vspace{0.2 cm}

        \begin{twocolentry}{
            \href{https://github.com/bartongroup/FRAGSYS}{FRAGSYS}
        }
            \textbf{FRAGSYS}
            \begin{highlights}
                \item A pipeline for the analysis of fragment screening-derived binding sites
                \item Tools: Python, Bash, Jalview, OC
                \item Packages: Matplotlib, Pandas, Seaborn
            \end{highlights}
        \end{twocolentry}


        \vspace{0.2 cm}

        \begin{twocolentry}{
            \href{https://github.com/bartongroup/ANK-analysis}{ANK-analysis}
        }
            \textbf{ANK-analysis}
            \begin{highlights}
                \item The most comprehensive analysis of Ankyrin repeats
                \item Tools: Python, ClustalO, Chimera, AMPS, gnomAD
                \item Packages: ProIntVar, ProteoFAV, VarAlign
            \end{highlights}
        \end{twocolentry}



    
    \section{Conferences}



        
        \begin{samepage}
            \begin{twocolentry}{
                Mar 2025
            }
                \textbf{ELIXIR 3DBioInfo Annual General Meeting | ISCB 3DSIG}

                \vspace{0.10 cm}

                \mbox{Talk: \textit{"Computational methods for the characterisation and evaluation of protein-ligand  binding sites"}}
                \vspace{0.10 cm}

        Barcelona, Spain    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                July 2024
            }
                \textbf{ISCB ISMB 2024}

                \vspace{0.10 cm}

                \mbox{Poster: \textit{"LIGYSIS: a resource for the analysis of ligand binding sites"}}
                \vspace{0.10 cm}

        Montreal, Canada    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Nov 2023
            }
                \textbf{3DBioInfo | ICSB 3DSIG | ELIXIR Czech Republic Community Meeting}

                \vspace{0.10 cm}

                \mbox{Poster: \textit{"LIGYSIS: a resource for the analysis of ligand binding sites"}}
                \vspace{0.10 cm}

        Prague, Czech Republic    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                July 2023
            }
                \textbf{ISCB ECCB/ISMB 2023}

                \vspace{0.10 cm}

                \mbox{Poster: \textit{"Characterising the human protein-ligand interactome"}}
                \vspace{0.10 cm}

        Lyon, France    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Mar 2023
            }
                \textbf{EMBO VizBi 2023}

                \vspace{0.10 cm}

                \mbox{Poster + flash talk: \textit{"Predicting function in ligand binding sites"}}
                \vspace{0.10 cm}

        Heidelberg, Germany    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                Sept 2022
            }
                \textbf{ISCB ECCB 2022}

                \vspace{0.10 cm}

                \mbox{Poster: \textit{"Fragment screening in context with human population variation"}}
                \vspace{0.10 cm}

        Sitges, Spain    \end{twocolentry}
        \end{samepage}

        \vspace{0.2 cm}

        \begin{samepage}
            \begin{twocolentry}{
                June 2021
            }
                \textbf{European Society for Evolutionary Biology Symposium}

                \vspace{0.10 cm}

                \mbox{Talk: \textit{"Tandem Repeats: methods and roles in molecular evolution"}}
                \vspace{0.10 cm}

        Online    \end{twocolentry}
        \end{samepage}


    
    \section{Assets}



        
        \begin{onecolentry}
            \textbf{Languages:} Python, Bash, R, JavaScript, HTML, CSS.
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{Software:} Jalview, ChimeraX, PyMol, Clustal, HMMER.
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{Skills:} Protein sequence and structure analysis, machine learning, web development, workflow generation, method benchmarking, version control, scientific writing, supervision, team work.
        \end{onecolentry}


    
    \section{Languages}



        
        \begin{onecolentry}
            \textbf{Spanish and Catalan:} Mother tongue
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{English:} Advanced level, C1 (Grade A, First Certificate in English)
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{German:} Advanced level, B2 (5 years at Official Language School)
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{French:} Basic level
        \end{onecolentry}


    
    \section{References}



        
        \begin{onecolentry}
            \textbf{PhD Supervisor:} \href{https://www.dundee.ac.uk/people/geoffrey-barton}{Prof Geoff Barton}
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{MSc Secondary Supervisor:} \href{https://www.dundee.ac.uk/people/stuart-macgowan}{Dr Stuart MacGowan}
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{PIPS Supervisor:} \href{https://www.linkedin.com/in/andrew-croydon-55623a14b/}{Dr Andrew Croydon}
        \end{onecolentry}

        \vspace{0.2 cm}

        \begin{onecolentry}
            \textbf{BSc Dissertation Supervisor:} \href{https://ibb.uab.cat/wp-content/themes/viral/modules/ibb_membres/view_membre.php?CodiMembre=224}{Prof Antonio Barbadilla}
        \end{onecolentry}


    

\end{document}