# Dr Javier Sánchez Utgés's CV

- Phone: +44 7878 459945
- Email: [utges03@gmail.com](mailto:utges03@gmail.com)
- ORCID: 
    [0000-0002-1078-1440](https://orcid.org/0000-0002-1078-1440)
- LinkedIn: 
    [Javier Sánchez Utgés](https://www.linkedin.com/in/javier-sanchez-utges-6aa2961b3/)
- GitHub: 
    [JavierSanchez-Utges](https://github.com/JavierSanchez-Utges)
- Google Scholar: 
    [Javier S Utgés](https://scholar.google.com/citations?user=YOUR_USER_ID)


# Education

## University of Dundee, PhD in Structural Bioinformatics

- Sept 2020 – Mar 2025
- Supervised 7 students across BSc, MSc and PhD degrees
- Student Representative of Computational Biology Division
- Thesis: *"Computational methods for the characterisation and evaluation of protein-ligand binding sites"*

## Universitat Pompeu Fabra, MSc in Bioinformatics for Health Sciences

- Sept 2018 – July 2020
- Grade: 9.3/10
- 9-month Internship at the [Barton Group](https://www.compbio.dundee.ac.uk/), University of Dundee
- Thesis: *"Ankyrin repeats in context with population variation"*

## Universitat Autònoma de Barcelona, BSc in Genetics

- Sept 2014 – July 2018
- Grade: 8.4/10
- 12-month Erasmus exchange at University of Dundee
- Dissertation: *“Methods for the detection and estimation of selection acting upon the genome”*

# Experience

## University of Dundee, Research assistant

- Mar 2025 – June 2025
- Dundee, Scotland

## The Association of the British Pharmaceutical Industry (ABPI), Intern

- Aug 2022 – Nov 2022
- London, England
- Improved the Interactive Resources for Schools website of the [ABPI](https://www.abpi.org.uk/)

## IBE-CSIC-UPF, Intern

- July 2017 – Sept 2017
- Barcelona, Spain
- Internship at the [Calafell Lab](https://www.ibe.upf-csic.es/calafell) on Forensic and Population Genetics

# Publications

## LIGYSIS-web: a resource for the analysis of protein-ligand binding sites ([10.1093/nar/gkaf411](https://doi.org/10.1093/nar/gkaf411))
- May 2025
- ***Utgés JS***, MacGowan SA, Barton GJ
- Nucleic Acids Research

## Comparative evaluation of methods for the prediction of protein-ligand binding sites ([10.1186/s13321-024-00923-z](https://doi.org/10.1186/s13321-024-00923-z))
- Nov 2024
- ***Utgés JS***, Barton GJ
- Journal of Cheminformatics

## Classification of likely functional class for ligand binding sites identified from fragment screening ([10.1038/s42003-024-05970-8](https://doi.org/10.1038/s42003-024-05970-8))
- Mar 2024
- ***Utgés JS***, MacGowan SA, Ives CM, Barton GJ
- Communications Biology

## Ankyrin repeats in context with human population variation ([10.1371/journal.pcbi.1009335](https://doi.org/10.1371/journal.pcbi.1009335))
- Aug 2021
- ***Utgés JS***, Tsenkov MI, Dietrich NJM, MacGowan SA, Barton GJ
- PLOS Computational Biology

# Projects

## LIGYSIS-web

- [LIGYSIS-web](https://github.com/JavierSanchez-Utges/LIGYSIS-web)
- A Python Flask Web Application to explore protein-ligand binding sites
- Tools: Python, JavaScript, HTML, CSS
- Packages: Flask, 3DMol.js, Chart.js

## LBS-comparison

- [LBS-comparison](https://github.com/bartongroup/LBS-comparison)
- The largest benchmark of ligand binding site prediction methods.
- Tools: Python, P2Rank, fpocket, IF-SitePred, POVME
- Packages: BioPython, Pandas, Sklearn

## FRAGSYS

- [FRAGSYS](https://github.com/bartongroup/FRAGSYS)
- A pipeline for the analysis of fragment screening-derived binding sites
- Tools: Python, Bash, Jalview, OC
- Packages: Matplotlib, Pandas, Seaborn

## ANK-analysis

- [ANK-analysis](https://github.com/bartongroup/ANK-analysis)
- The most comprehensive analysis of Ankyrin repeats
- Tools: Python, ClustalO, Chimera, AMPS, gnomAD
- Packages: ProIntVar, ProteoFAV, VarAlign

# Conferences

## ELIXIR 3DBioInfo Annual General Meeting | ISCB 3DSIG 
- Mar 2025
- Talk: *"Computational methods for the characterisation and evaluation of protein-ligand  binding sites"*
- Barcelona, Spain

## ISCB ISMB 2024 
- July 2024
- Poster: *"LIGYSIS: a resource for the analysis of ligand binding sites"*
- Montreal, Canada

## 3DBioInfo | ICSB 3DSIG | ELIXIR Czech Republic Community Meeting 
- Nov 2023
- Poster: *"LIGYSIS: a resource for the analysis of ligand binding sites"*
- Prague, Czech Republic

## ISCB ECCB/ISMB 2023 
- July 2023
- Poster: *"Characterising the human protein-ligand interactome"*
- Lyon, France

## EMBO VizBi 2023 
- Mar 2023
- Poster + flash talk: *"Predicting function in ligand binding sites"*
- Heidelberg, Germany

## ISCB ECCB 2022 
- Sept 2022
- Poster: *"Fragment screening in context with human population variation"*
- Sitges, Spain

## European Society for Evolutionary Biology Symposium 
- June 2021
- Talk: *"Tandem Repeats: methods and roles in molecular evolution"*
- Online

# Assets

- Languages: Python, Bash, R, JavaScript, HTML, CSS.
- Software: Jalview, ChimeraX, PyMol, Clustal, HMMER.
- Skills: Protein sequence and structure analysis, machine learning, web development, workflow generation, method benchmarking, version control, scientific writing, supervision, team work.
# Languages

- Spanish and Catalan: Mother tongue
- English: Advanced level, C1 (Grade A, First Certificate in English)
- German: Advanced level, B2 (5 years at Official Language School)
- French: Basic level
# References

- PhD Supervisor: [Prof Geoff Barton](https://www.dundee.ac.uk/people/geoffrey-barton)
- MSc Secondary Supervisor: [Dr Stuart MacGowan](https://www.dundee.ac.uk/people/stuart-macgowan)
- PIPS Supervisor: [Dr Andrew Croydon](https://www.linkedin.com/in/andrew-croydon-55623a14b/)
- BSc Dissertation Supervisor: [Prof Antonio Barbadilla](https://ibb.uab.cat/wp-content/themes/viral/modules/ibb_membres/view_membre.php?CodiMembre=224)
