<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>
        Dr Javier Sánchez Utgés's CV
    </title>
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/5.5.1/github-markdown-light.min.css"
        integrity="sha512-Pmhg2i/F7+5+7SsdoUqKeH7UAZoVMYb1sxGOoJ0jWXAEHP0XV2H4CITyK267eHWp2jpj7rtqWNkmEOw1tNyYpg=="
        crossorigin="anonymous" referrerpolicy="no-referrer" />
    <style>
        .markdown-body {
            box-sizing: border-box;
            min-width: 200px;
            max-width: 980px;
            margin: 0 auto;
            padding: 45px;
        }

        @media (max-width: 767px) {
            .markdown-body {
                padding: 15px;
            }
        }
    </style>
</head>

<body>
    <article class="markdown-body">
        <h1>Dr Javier Sánchez Utgés's CV</h1>
        <ul>
        <li>Phone: +44 7878 459945</li>
        <li>Email: <a href="mailto:utges03@gmail.com">utges03@gmail.com</a></li>
        <li>ORCID: 
            <a href="https://orcid.org/0000-0002-1078-1440">0000-0002-1078-1440</a></li>
        <li>LinkedIn: 
            <a href="https://www.linkedin.com/in/javier-sanchez-utges-6aa2961b3/">Javier Sánchez Utgés</a></li>
        <li>GitHub: 
            <a href="https://github.com/JavierSanchez-Utges">JavierSanchez-Utges</a></li>
        <li>Google Scholar: 
            <a href="https://scholar.google.com/citations?user=YOUR_USER_ID">Javier S Utgés</a></li>
        </ul>
        <h1>Education</h1>
        <h2>University of Dundee, PhD in Structural Bioinformatics</h2>
        <ul>
        <li>Sept 2020 – Mar 2025</li>
        <li>Supervised 7 students across BSc, MSc and PhD degrees</li>
        <li>Student Representative of Computational Biology Division</li>
        <li>Thesis: <em>"Computational methods for the characterisation and evaluation of protein-ligand binding sites"</em></li>
        </ul>
        <h2>Universitat Pompeu Fabra, MSc in Bioinformatics for Health Sciences</h2>
        <ul>
        <li>Sept 2018 – July 2020</li>
        <li>Grade: 9.3/10</li>
        <li>9-month Internship at the <a href="https://www.compbio.dundee.ac.uk/">Barton Group</a>, University of Dundee</li>
        <li>Thesis: <em>"Ankyrin repeats in context with population variation"</em></li>
        </ul>
        <h2>Universitat Autònoma de Barcelona, BSc in Genetics</h2>
        <ul>
        <li>Sept 2014 – July 2018</li>
        <li>Grade: 8.4/10</li>
        <li>12-month Erasmus exchange at University of Dundee</li>
        <li>Dissertation: <em>“Methods for the detection and estimation of selection acting upon the genome”</em></li>
        </ul>
        <h1>Experience</h1>
        <h2>University of Dundee, Research assistant</h2>
        <ul>
        <li>Mar 2025 – June 2025</li>
        <li>Dundee, Scotland</li>
        </ul>
        <h2>The Association of the British Pharmaceutical Industry (ABPI), Intern</h2>
        <ul>
        <li>Aug 2022 – Nov 2022</li>
        <li>London, England</li>
        <li>Improved the Interactive Resources for Schools website of the <a href="https://www.abpi.org.uk/">ABPI</a></li>
        </ul>
        <h2>IBE-CSIC-UPF, Intern</h2>
        <ul>
        <li>July 2017 – Sept 2017</li>
        <li>Barcelona, Spain</li>
        <li>Internship at the <a href="https://www.ibe.upf-csic.es/calafell">Calafell Lab</a> on Forensic and Population Genetics</li>
        </ul>
        <h1>Publications</h1>
        <h2>LIGYSIS-web: a resource for the analysis of protein-ligand binding sites (<a href="https://doi.org/10.1093/nar/gkaf411">10.1093/nar/gkaf411</a>)</h2>
        <ul>
        <li>May 2025</li>
        <li><strong><em>Utgés JS</em></strong>, MacGowan SA, Barton GJ</li>
        <li>Nucleic Acids Research</li>
        </ul>
        <h2>Comparative evaluation of methods for the prediction of protein-ligand binding sites (<a href="https://doi.org/10.1186/s13321-024-00923-z">10.1186/s13321-024-00923-z</a>)</h2>
        <ul>
        <li>Nov 2024</li>
        <li><strong><em>Utgés JS</em></strong>, Barton GJ</li>
        <li>Journal of Cheminformatics</li>
        </ul>
        <h2>Classification of likely functional class for ligand binding sites identified from fragment screening (<a href="https://doi.org/10.1038/s42003-024-05970-8">10.1038/s42003-024-05970-8</a>)</h2>
        <ul>
        <li>Mar 2024</li>
        <li><strong><em>Utgés JS</em></strong>, MacGowan SA, Ives CM, Barton GJ</li>
        <li>Communications Biology</li>
        </ul>
        <h2>Ankyrin repeats in context with human population variation (<a href="https://doi.org/10.1371/journal.pcbi.1009335">10.1371/journal.pcbi.1009335</a>)</h2>
        <ul>
        <li>Aug 2021</li>
        <li><strong><em>Utgés JS</em></strong>, Tsenkov MI, Dietrich NJM, MacGowan SA, Barton GJ</li>
        <li>PLOS Computational Biology</li>
        </ul>
        <h1>Projects</h1>
        <h2>LIGYSIS-web</h2>
        <ul>
        <li><a href="https://github.com/JavierSanchez-Utges/LIGYSIS-web">LIGYSIS-web</a></li>
        <li>A Python Flask Web Application to explore protein-ligand binding sites</li>
        <li>Tools: Python, JavaScript, HTML, CSS</li>
        <li>Packages: Flask, 3DMol.js, Chart.js</li>
        </ul>
        <h2>LBS-comparison</h2>
        <ul>
        <li><a href="https://github.com/bartongroup/LBS-comparison">LBS-comparison</a></li>
        <li>The largest benchmark of ligand binding site prediction methods.</li>
        <li>Tools: Python, P2Rank, fpocket, IF-SitePred, POVME</li>
        <li>Packages: BioPython, Pandas, Sklearn</li>
        </ul>
        <h2>FRAGSYS</h2>
        <ul>
        <li><a href="https://github.com/bartongroup/FRAGSYS">FRAGSYS</a></li>
        <li>A pipeline for the analysis of fragment screening-derived binding sites</li>
        <li>Tools: Python, Bash, Jalview, OC</li>
        <li>Packages: Matplotlib, Pandas, Seaborn</li>
        </ul>
        <h2>ANK-analysis</h2>
        <ul>
        <li><a href="https://github.com/bartongroup/ANK-analysis">ANK-analysis</a></li>
        <li>The most comprehensive analysis of Ankyrin repeats</li>
        <li>Tools: Python, ClustalO, Chimera, AMPS, gnomAD</li>
        <li>Packages: ProIntVar, ProteoFAV, VarAlign</li>
        </ul>
        <h1>Conferences</h1>
        <h2>ELIXIR 3DBioInfo Annual General Meeting | ISCB 3DSIG</h2>
        <ul>
        <li>Mar 2025</li>
        <li>Talk: <em>"Computational methods for the characterisation and evaluation of protein-ligand  binding sites"</em></li>
        <li>Barcelona, Spain</li>
        </ul>
        <h2>ISCB ISMB 2024</h2>
        <ul>
        <li>July 2024</li>
        <li>Poster: <em>"LIGYSIS: a resource for the analysis of ligand binding sites"</em></li>
        <li>Montreal, Canada</li>
        </ul>
        <h2>3DBioInfo | ICSB 3DSIG | ELIXIR Czech Republic Community Meeting</h2>
        <ul>
        <li>Nov 2023</li>
        <li>Poster: <em>"LIGYSIS: a resource for the analysis of ligand binding sites"</em></li>
        <li>Prague, Czech Republic</li>
        </ul>
        <h2>ISCB ECCB/ISMB 2023</h2>
        <ul>
        <li>July 2023</li>
        <li>Poster: <em>"Characterising the human protein-ligand interactome"</em></li>
        <li>Lyon, France</li>
        </ul>
        <h2>EMBO VizBi 2023</h2>
        <ul>
        <li>Mar 2023</li>
        <li>Poster + flash talk: <em>"Predicting function in ligand binding sites"</em></li>
        <li>Heidelberg, Germany</li>
        </ul>
        <h2>ISCB ECCB 2022</h2>
        <ul>
        <li>Sept 2022</li>
        <li>Poster: <em>"Fragment screening in context with human population variation"</em></li>
        <li>Sitges, Spain</li>
        </ul>
        <h2>European Society for Evolutionary Biology Symposium</h2>
        <ul>
        <li>June 2021</li>
        <li>Talk: <em>"Tandem Repeats: methods and roles in molecular evolution"</em></li>
        <li>Online</li>
        </ul>
        <h1>Assets</h1>
        <ul>
        <li>Languages: Python, Bash, R, JavaScript, HTML, CSS.</li>
        <li>Software: Jalview, ChimeraX, PyMol, Clustal, HMMER.</li>
        <li>Skills: Protein sequence and structure analysis, machine learning, web development, workflow generation, method benchmarking, version control, scientific writing, supervision, team work.</li>
        </ul>
        <h1>Languages</h1>
        <ul>
        <li>Spanish and Catalan: Mother tongue</li>
        <li>English: Advanced level, C1 (Grade A, First Certificate in English)</li>
        <li>German: Advanced level, B2 (5 years at Official Language School)</li>
        <li>French: Basic level</li>
        </ul>
        <h1>References</h1>
        <ul>
        <li>PhD Supervisor: <a href="https://www.dundee.ac.uk/people/geoffrey-barton">Prof Geoff Barton</a></li>
        <li>MSc Secondary Supervisor: <a href="https://www.dundee.ac.uk/people/stuart-macgowan">Dr Stuart MacGowan</a></li>
        <li>PIPS Supervisor: <a href="https://www.linkedin.com/in/andrew-croydon-55623a14b/">Dr Andrew Croydon</a></li>
        <li>BSc Dissertation Supervisor: <a href="https://ibb.uab.cat/wp-content/themes/viral/modules/ibb_membres/view_membre.php?CodiMembre=224">Prof Antonio Barbadilla</a></li>
        </ul>
    </article>
</body>

</html>